src/main/java/org/sbx/file/parser/Parser.java

```diff
@@ -0,0 +1,82 @@
+package org.sbx.file.parser;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.sbx.messages.impl.ApplicationDebugMessage;
+
+import java.text.DateFormat;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * Created by aloginov on 01.11.16.
+ */
+public class Parser {
+    private String targetString;
+    private String regExp;
+    private List<String> list;
+
+    private static final Logger logger = LogManager.getLogger(Parser.class);
+    public Date getDateByString(String stringDate, String format){
+        Date date = null;
+        DateFormat dateFormat = new SimpleDateFormat(format); //MM.dd.yyyy HH:mm:ss.SSS
```

Comment:
Leave comments above , not from the right.
Try to leave more clear and detailed comments